orbit/pkg/table/software_update/software_update_test.go

```diff
@@ -0,0 +1,40 @@
+//go:build darwin
```

Comment:
FYI: Test looks great!, though currently `go test`s only run on `ubuntu-latest` only:
https://github.com/fleetdm/fleet/blob/995f2781149a6f48cf831ea0fe19a8a005673d91/.github/workflows/test-go.yaml#L38-L45
There's some tech debt to also run them in `macos-latest` too.

Reply:
👍 
We definitely need to run them on Mac as well.